### PR TITLE
Added a composer.json to enable support for PIE

### PR DESCRIPTION
**Pull request by asgrim**
Adds support for [PIE](https://github.com/php/pie).

Once merged, a maintainer would need to add this repo's URL to https://packagist.org/packages/submit :)

More info/documentation can be read on: https://github.com/php/pie/blob/main/docs/extension-maintainers.md

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+    "name": "php-memcached/php-memcached",
```
Comment (asgrim) [inline composer.json:2]: Please verify vendor namespace & package name are acceptable

**Comment by TimWolla** [comment]
> Once merged, a maintainer would need to add this repo's URL to https://packagist.org/packages/submit :)

It appears this has not yet happened.

**Comment by sodabrew** [comment]
Done! https://packagist.org/packages/php-memcached/php-memcached

**Comment by TimWolla** [comment]
@sodabrew Thank you. I notice that Packagist says “This package is not [auto-updated](https://packagist.org/about#how-to-update-packages)”. You would then need to remember to update the Packagist data for every release. It would probably make sense to also set up the webhook.

**Comment by asgrim** [comment]
hi @sodabrew any chance the auto update could be set up as mentioned above? :) would certainly make things smoother for future releases :) thank you!

**Comment by asgrim** [comment]
Ping on the above please! :)

**Comment by m6w6** [comment]
@sodabrew , since you own the package on packagist, could you please update the webhook?

**Comment by sodabrew** [comment]
@m6w6 done!

**Comment by TimWolla** [comment]
> since you own the package on packagist

FYI: It's also possible to add additional maintainers to Packagist entry. The large “plus” in the screenshot should work.
image

**Pull request merged**